docs/release-procedure.md

```diff
@@ -0,0 +1,44 @@
+# Release Procedure
```

Comment:
I didn't have chance to try this procedure , @dulek I assume you will try this or already tried?

Reply:
Yes, I made releases a few times. That's why I'm adding back the point about updating `images.yaml` in `kubernetes/k8s.io`, because that one was have been removed by mistake.